index.html

```diff
@@ -0,0 +1,198 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <!-- <link rel="stylesheet" href="/style/desktopAbout.css"> -->
+  <link rel="stylesheet" href="/style/mobileIndex.css">
+  <link rel="stylesheet" href="/style/DesktopIndex.css">
```

Comment:
Kindly note that using absolute path doesn't work on GitHub Pages, this is the reason your styles are not being loaded in the live version. Kindly replace it with a relative path - that should fix the problem 😃
```href="./style/mobileIndex.css"```

![image](https://user-images.githubusercontent.com/73714615/191939651-cd191a14-a929-4c9a-bf4a-8c598a472e06.png)